src/util/types.ts

```diff
@@ -1,14 +1,14 @@
-import deepEqual from "deep-equal";
 import {Type} from "@chainsafe/ssz";
-import {types as mainnetTypes} from "@chainsafe/lodestar-types/lib/ssz/presets/mainnet";
-import {types as minimalTypes} from "@chainsafe/lodestar-types/lib/ssz/presets/minimal";
+import * as sszTypes from "@chainsafe/lodestar-types/lib/sszTypes";
 
-export const presets = {
-  mainnet: mainnetTypes as unknown as Record<string, Type<any>>,
-  minimal: minimalTypes as unknown as Record<string, Type<any>>,
-} as const;
+const {phase0, altair, allForks, ...primitive} = sszTypes;
 
-export type PresetName = keyof typeof presets;
+export const forks = {
+  phase0: {...phase0, ...primitive} as Record<string, Type<unknown>>,
+  altair: {...altair, ...primitive} as Record<string, Type<unknown>>,
```

Comment:
Can you set altair to `{...phase0, ...altair, ...primitive}`
This should also add all phase0 types that aren't updated in altair.